WordPress/src/main/java/org/wordpress/android/ui/reader/ReaderPostListFragment.java

```diff
@@ -812,6 +814,28 @@ private void updateCurrentTag() {
         updatePostsWithTag(getCurrentTag(), UpdateAction.REQUEST_NEWER);
     }
 
+    /*
+     * update the current tag if it's time to do so
+     */
+    private void updateCurrentTagIfTime() {
+        if (!isAdded() || !hasCurrentTag()) {
+            return;
+        }
+        new Thread() {
```

Comment:
Can we add a comment here to explain why we need this (eventually with a ref to that PR).